pkg/commands/app.go

```diff
@@ -517,6 +518,24 @@ func NewRootfsCommand() *cli.Command {
 	}
 }
 
+// NewSbomCommand is the factory method to add sbom command
+func NewSbomCommand() *cli.Command {
+	return &cli.Command{
+		Name:      "sbom",
+		ArgsUsage: "artifact",
+		Usage:     "generate sbom for an artifact",
+		Action:    artifact.SbomRun,
+		Flags: []cli.Flag{
+			&formatFlag,
+			&outputFlag,
+			&exitCodeFlag,
```

Comment:
Looks like `sbom` doesn't need `--exit-code` since this option makes the job fail when critical vulnerabilities are found.